components/user_settings/modal/user_settings_modal.jsx

```diff
@@ -110,10 +110,14 @@ class UserSettingsModal extends React.Component {
         document.removeEventListener('keydown', this.handleKeyDown);
     }
 
-    componentDidUpdate() {
+    componentDidUpdate(_prevProps, prevState) {
```

Comment:
I get the point of using the _ to represent that you are not using that parameter at all, but I think we don't use that in any other place, so I would prefer to simply use `prevProps`.